source/Server.Contracts/Endpoints/DeploymentActionContainerResource.cs

```diff
@@ -1,8 +1,10 @@
+using Octopus.Server.MessageContracts.Features.Feeds;
+
 namespace Sashimi.Server.Contracts.Endpoints
 {
     public class DeploymentActionContainerResource
     {
         public string? Image { get; set; }
-        public string? FeedId { get; set; }
```

Comment:
I've left this as `FeedId` (rather than changing to `FeedIdOrName`) because I found examples of other resources that had been changed to TinyTypes that kept the property name as-is.